README.md

```diff
@@ -244,14 +244,15 @@ a simple example for props would look like this
 
 note that property value type should be **numeric**(byte, short, integer, float, double) or **boolean** or **string**.
 
-**indexProps** define primary index for this label(like `PRIMARY INDEX idx_xxx`(`p1, p2`) in RDBMS).
+**indices** define indices for this label
 
-s2graph will automatically keep edges sorted according to this indexProps. 
+first index in indices is primary (like `PRIMARY INDEX idx_xxx`(`p1, p2`) in RDBMS).
 
-extra indexProps can be defined later when need multiple ordering on edges.
+s2graph will automatically keep edges sorted according to this index.
 
-**props** define meta datas that will not be affect the order of edges. 
+other's for multiple ordering on edges.(like `ALTER TABLE ADD INDEX idx_xxx`(`p2, p1`) in RDBMS).
```

Comment:
changing alter syntax into index syntax in create table.(`INDEX `idx_xxx` (`p2, p1`) in RDBMS)